examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -25,8 +25,48 @@ const app = {
       this.onDeviceReady.bind(this),
       false
     );
+    // this method gets called after configure is done
```

Comment:
It also gets called after a successful purchase, or after `Purchases.getCustomerInfo` brings a new `customerInfo` right? So it will execute everything again in all those cases

Reply:
good call, I'll update so it gets called just the one time